jboss-kie-kieserver/added/launch/kieserver-pull.sh

```diff
@@ -31,7 +31,7 @@ function generatePullPomXml() {
 </project>"
 }
 
-if [ "${KIE_SERVER_CONTAINER_DEPLOYMENT}" != "" ]; then
+if [[ "x${KIE_SERVER_CONTAINER_DEPLOYMENT}" != "x" && "${KIE_SERVER_DISABLE_KC_PULL_DEPS^^}" != "TRUE" ]]; then
```

Comment:
Is KIE_SERVER_DISABLE_KC_PULL_DEPS correctly hand over to this script? In jboss-kie-server.sh is to file only handed over the verification env var. See https://github.com/jboss-container-images/jboss-kie-modules/pull/438/files#diff-3e7628a0c0671f1c47f2f4da279754577748d791191318fb85c3d250b1ca182cL689

Reply:
yes, that was a leftover.